src/chains/tests/test_models.py

```diff
@@ -10,3 +13,28 @@ def test_str_method_outputs_name_chain_id(self):
             str(chain),
             f"{chain.name} | chain_id={chain.id}",
         )
+
+
+class ChainColorValidationTestCase(TransactionTestCase):
+    def test_invalid_text_colors(self):
+        param_list = ["aaa", "bbb", "#fffffffff", "zzz", "010", "", "a word", "#hhh"]
+        for invalid_color in param_list:
+            with self.subTest(msg=f"Invalid color {invalid_color} should throw"):
+                with self.assertRaises(
+                    (
+                        ValidationError,
+                        DataError,
+                    )
+                ):
+                    chain = ChainFactory.create(theme_text_color=invalid_color)
+                    # run validators
+                    chain.full_clean()
+
+    def test_valid_text_colors(self):
+        param_list = ["#000", "#fff", "#00000000", "#ffffffff"] + [
+            Faker().hex_color() for _ in range(20)
+        ]
```

Comment:
Added some valid edge cases + some additional random 20 ones that should pass the test